### PR TITLE
refactor: move translations plugin to npm

### LINKED ISSUE #6138: move translations plugin to its own npm package, and install npm version Reaction
New repo: https://github.com/reactioncommerce/plugin-translations  

Convert the translations plugin. This includes moving all related files into a new repository, releasing that repo as an `npm` package, and then updating this repo, `reaction`, to use the `npm` version of the plugin.

Part of [this epic](https://github.com/reactioncommerce/reaction/issues/6107) to move all plugins into their own repositories and release as `npm` packages.

### PR DESCRIPTION
**Pull request by kieckhafer**
Resolves #6138  

This PR removes the internal `translations` plugin, and installs the new `plugin-translations` as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

### CODE REVIEW

**Comment by kieckhafer** [comment]
@chrispotter Snyk is throwing an error because the new plugin is GPL. It is OK to merge if that's the only issue, we'll need to add it to an exception list *after* it's been merged.

**Review by chrispotter — APPROVED** [review]

**Pull request merged**
